spec/menu_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'menu'
+
+describe Menu do
+  let(:menu) { Menu.new }
+
+  it 'should respond to view' do
+    expect(menu).to respond_to(:view_menu)
+  end
+
+  it 'should print the menu' do
+    expect(menu.view_menu).to eq({ "Apple" => 1, "Banana" => 2, "Mango" => 3, "Peach" => 4, "Pear" => 5 })
```

Comment:
Ideally might not have the outcome hard-coded into the expected result. Could use a constant somewhere for various tests or maybe a double.